base-images/build.sh

```diff
@@ -15,22 +15,22 @@ fi
 # https://github.com/docker/buildx/issues/495#issuecomment-754688157
 docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
 
-docker buildx build --build-arg IMAGE_NAME=docker.io/nvidia/cuda \
+docker buildx build \
     --build-arg ENVD_VERSION=${ENVD_VERSION} \
     --build-arg ENVD_SSH_IMAGE=ghcr.io/tensorchord/envd-ssh-from-scratch \
     --build-arg HTTP_PROXY=${HTTP_PROXY} \
-     --build-arg HTTPS_PROXY=${HTTPS_PROXY} \
+    --build-arg HTTPS_PROXY=${HTTPS_PROXY} \
     --pull --push --platform linux/x86_64,linux/arm64 \
-    -t ${DOCKER_HUB_ORG}/python:3.8-ubuntu20.04-cuda11.6-cudnn8 \
-    -f python3.8-ubuntu20.04-cuda11.6.Dockerfile .
-docker buildx build \
+    -t ${DOCKER_HUB_ORG}/python:3.9-ubuntu20.04 \
+    -f python3.9-ubuntu20.04.Dockerfile .
```

Comment:
Consider parameterizing these to avoid redundant typing and reduce maintenance effort.

Reply:
I think we should create a new repo and automate the base image-building process.